aten/src/ATen/native/cuda/Math.cuh

```diff
@@ -88,5 +88,19 @@ static inline __host__ __device__ scalar_t calc_trigamma(scalar_t in) {
   return static_cast<scalar_t>(sign * result);
 }
 
+template <typename scalar_t>
+static inline __host__ __device__ scalar_t calc_gcd(scalar_t a, scalar_t b) {
```

Comment:
You can use: 

https://github.com/pytorch/pytorch/blob/a2ef54c598261ad9fd15e1e85c71f15055652572/c10/macros/Macros.h#L157

Reply:
All other functions in that header file also don't use that macro. Should I change them too?

Reply:
No. Old code that's working is best left undisturbed. Thank you for asking, though.